build-requirements.txt

```diff
@@ -1 +1 @@
-charmcraft==0.9.0
+git+https://github.com/canonical/charmcraft.git@0.10.2#egg=charmcraft
```

Comment:
Do we need this anymore? It seems carry over from a previous model? I think we've dropped this from various locations as we use the charmcraft snap. In fact, refer to the cinder-storage-backend-template project for this as well.